packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -139,157 +148,91 @@ export class Telemetry {
     Telemetry.setupClient();
 
     await Telemetry.setupBaseProperties();
+  }
+
+  private static basicTelemetryInitializer(envelope: coreOneDS.ITelemetryItem) : boolean
+  {    
+    // Filter out "legacy" events from older stable branches
+    if (envelope.name && EventNamesWeTrack.includes(envelope.name)) {
+      return true;
+    }
 
-    Telemetry.setupTelemetryProcessors();
+    return false;
   }
 
-  /** Sets up Telemetry.client. */
+  /** Sets up Telemetry.appInsightsCore. */
   private static setupClient() {
-    appInsights.Configuration.setInternalLogging(false, false);
-
-    Telemetry.client = new appInsights.TelemetryClient(
-      Telemetry.options.setupString,
-    );
+    const postChannel: PostChannel = new PostChannel();
 
-    // Allow overriding the proxy server via an environment variable
-    const proxyServer = process.env[ENV_PROXY_OVERRIDE];
-    if (proxyServer) {
-      Telemetry.client.config.proxyHttpUrl = proxyServer;
-      Telemetry.client.config.proxyHttpsUrl = proxyServer;
+    const coreConfiguration: coreOneDS.IExtendedConfiguration = {
+      instrumentationKey: process.env[ENV_SETUP_OVERRIDE] ?? RNW_1DS_INSTRUMENTATION_KEY,
     }
 
-    Telemetry.client.config.disableAppInsights = Telemetry.isTest;
-    Telemetry.client.config.disableStatsbeat = true;
+    const postChannelConfig: IChannelConfiguration = {
+      eventsLimitInMem: 5000
+    };
 
-    // Despite trying to disable the statsbeat, it might still be running: https://github.com/microsoft/ApplicationInsights-node.js/issues/943
-    // So we want to disable it, and despite the method's typing, getStatsbeat() _can_ return undefined
+    coreConfiguration.extensionConfig = {};
+    coreConfiguration.extensionConfig[postChannel.identifier] = postChannelConfig;
 
-    Telemetry.client.getStatsbeat()?.enable(false);
+    // Allow overriding the endpoint URL via an environment variable.
+    if (process.env[ENV_PROXY_OVERRIDE] !== undefined) {
+      coreConfiguration.endpointUrl = process.env[ENV_PROXY_OVERRIDE];
+    }
+  
+    Telemetry.appInsightsCore = new coreOneDS.AppInsightsCore();
+    Telemetry.appInsightsCore.initialize(coreConfiguration, [postChannel] /* extensions */);
 
-    Telemetry.client.channel.setUseDiskRetryCaching(!Telemetry.isTest);
+    Telemetry.appInsightsCore.addTelemetryInitializer(Telemetry.basicTelemetryInitializer);
   }
 
   /** Sets up any base properties that all telemetry events require. */
   private static async setupBaseProperties() {
-    Telemetry.client!.commonProperties.deviceId =
+    Telemetry.commonProperties.deviceId =
```

Comment:
A lot of these "common" properties were added by us when we couldn't find them in the AI payloads. But since 1DS follows Common Schema, maybe they're already available in Part A?

Reply:
Some of these props are available in Part A "extensions", but not all of them. I can check how the payload would look like if we use Part A extensions.

Reply:
I placed 4 of the common props into Part A extensions.